Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -879,3 +879,52 @@
     blacklist:
       tags:
       - Slice #no cheating!
+
+- type: cargoBounty
+  id: BountyCrystal
+  reward: 10000
+  description: bounty-description-crystal
+  entries:
+  - name: bounty-item-crystal
+    amount: 12
+    whitelist:
+      tags:
+      - CrystalRed
+      - CrystalPink
+      - CrystalOrange
+      - CrystalGreen
+      - CrystalCyan
+      - CrystalBlue
+
+- type: cargoBounty
+  id: BountyMimeHardsuit
+  reward: 13000
+  description: bounty-description-mime-hardsuit
+  entries:
+  - name: bounty-item-mime-hardsuit
+    amount: 1
+    whitelist:
+      tags:
+      - MimeHardsuit
+
+- type: cargoBounty
+  id: BountyTelecrystal #might get removed eventually teehee
```

Comment:
this is really just a slot filler, nobodys going to want to do this bounty except maybe thieves with useless tc. it either forces the skip timer or just sits there forever because nobody is available to skip it, so it becomes a dud

Reply:
cargo begs for Renault to die for no reason in particular

Reply:
I can get rid of it if we're sure, but I personally do like the idea of a traitor-only bounty. Plus, it shouldn't be showing up *that* often since I'm adding several more bounties to the pool

Reply:
Maybe amp up the price significantly to make it more attractive?

Reply:
if it shows up on lowpop with 0 qm its just going to be unskippable forever and limit cargo to 5/6. it might be uncommon, but its still going to happen and suck. its unobtainium unless you break rules and break into the bridge to kill renault

Reply:
I'll kill it then.